src/main.js

```diff
@@ -79,3 +84,33 @@ function updateColor() {
     timerButton.classList.add('exercise-button');
   }
 }
+
+function displayMinError() {
+  var minErrorMessage = document.querySelector('.min');
+  if (minInput.checkValidity() === false) {
+    minErrorMessage.classList.remove('hidden');
+    activityButton.disabled = true;
+  }
+}
+
+function displaySecError() {
+  var secErrorMessage = document.querySelector('.sec');
+  if (secInput.checkValidity() === false) {
+    secErrorMessage.classList.remove('hidden');
+    activityButton.disabled = true;
+  }
+}
+
+function displayDescError() {
+  var descErrorMessage = document.querySelector('.desc');
+  if (descInput.value === '') {
+    descErrorMessage.classList.remove('hidden');
+    activityButton.disabled = true;
+  }
+}
+
+function enableStartActivityButton() {
+  if (currentCategory !== '' && descInput.value !== '' && minInput.value !== '' && secInput.value !== '') {
```

Comment:
Love this function to check if these elements are not equal to empty strings to make the disabled button false.